py/selenium/webdriver/remote/webdriver.py

```diff
@@ -96,7 +97,13 @@ def _create_caps(caps):
     return {"capabilities": {"firstMatch": [{}], "alwaysMatch": always_match}}
 
 
-def get_remote_connection(capabilities, command_executor, keep_alive, ignore_local_proxy=False):
+def get_remote_connection(
+    capabilities: dict,
+    command_executor: Union[str, RemoteConnection],
```

Comment:
We have put `remote_addr` as deprecated but if people are using it to connect to a selenium grid this will be completed. Should we have if a `str` that we create a client config object for the user?

Reply:
@AutomatedTester, I added `if` statement below to create a ClientConfig in case command_executor is `str`
Can you check if it makes sense?